runtime/format/character.go

```diff
@@ -0,0 +1,29 @@
+/*
+ * Cadence - The resource-oriented smart contract programming language
+ *
+ * Copyright 2019-2021 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package format
+
+import (
+	"github.com/rivo/uniseg"
+)
+
+func IsValidCharacter(s string) bool {
```

Comment:
Good idea to add this function and re-use it👌 

Package `format` is so far meant for rendering Cadence values to strings. Maybe move the function to e.g. `sema` or `common`